### PR TITLE
docs: Theming guide

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Add theming guide. 

Screenshot 2024-10-14 at 5 00 32 PM

Screenshot 2024-10-14 at 5 00 43 PM

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/guides/theming.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: OnchainKit Themes · OnchainKit
+description: Customize the appearance of OnchainKit's components
+---
+
+# OnchainKit Themes
+{/* <img alt="Themes"
+  src=""
+  height="364"/> */}
```
Comment (cpcramer) [inline site/docs/pages/guides/theming.mdx:9]: TODO: Will update with GIF of updating themes

File: site/sidebar.ts [hunk 2]
```diff
@@ -9,32 +9,36 @@ export const sidebar = [
     text: 'Guides',
     items: [
       {
-        text: 'Lifecycle Status',
-        link: '/guides/lifecycle-status',
+        text: 'Build Onchain Apps',
+        link: '/guides/build-onchain-apps',
```
Comment (cpcramer) [inline site/sidebar.ts:13]: Made alphabetical

File: site/docs/pages/guides/framegear.mdx [hunk 3]
```diff
@@ -49,7 +49,7 @@ bun run dev
 Visit http://localhost:1337 to start the **Framegear** interface. Enter the URL of your locally
 running frame (e.g., `http://localhost:3000`) and click `Fetch` to validate your frame response and start testing.
 
-![Fetch Frame](https://onchainkit.xyz/assets/fetch-frame.png)]
+![Fetch Frame](https://onchainkit.xyz/assets/fetch-frame.png)
```
Comment (cpcramer) [inline site/docs/pages/guides/framegear.mdx:52]: typo fix

**Comment by cpcramer** [comment]
https://github.com/coinbase/onchainkit/blob/06d4fef6638bc5b8d499b4da073c4bbd5b591e66/site/docs/pages/getting-started.mdx?plain=1#L88-L92

**Review by abcrane123 — APPROVED** [review]

**Pull request merged**